var/spack/repos/builtin/packages/bat/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Reply:
Oops, shows when I started work on this.  Will fix....